src/Raven.Server/Documents/TcpHandlers/SubscriptionDocumentsFetcher.cs

```diff
@@ -82,6 +87,7 @@ public class SubscriptionDocumentsFetcher
                     0,
                     int.MaxValue))
                 {
+                    _threadAllocations = NativeMemory.CurrentThreadStats;
```

Comment:
Shouldn't this be _before_ the `foreach` ? This would cause the value to reset per iteration